astropy/cosmology/tests/test_cosmology.py

```diff
@@ -8,51 +8,51 @@
 
 import astropy.constants as const
 import astropy.units as u
-from astropy.cosmology import core, funcs, realizations
-from astropy.cosmology.realizations import WMAP5, WMAP7, WMAP9, Planck13, Planck15, Planck18
+from astropy.cosmology import Cosmology, flrw, funcs
+from astropy.cosmology.realizations import Planck13, Planck18, default_cosmology
 from astropy.units import allclose
 from astropy.utils.compat.optional_deps import HAS_SCIPY  # noqa
-from astropy.utils.exceptions import AstropyDeprecationWarning, AstropyUserWarning
+from astropy.utils.exceptions import AstropyUserWarning
 
 
 def test_init():
     """ Tests to make sure the code refuses inputs it is supposed to"""
     with pytest.raises(ValueError):
-        cosmo = core.FlatLambdaCDM(H0=70, Om0=-0.27)
+        cosmo = flrw.FlatLambdaCDM(H0=70, Om0=-0.27)
```

Comment:
How about just importing the actual classes? I don't know about others, but I often look at tests to see how code is used, and think it makes sense to follow some "best practice" for those - which I think would be either to use the top-level package or the class name directly.

Reply:
I agree with importing the actual classes, however I want to hold off on this PR because I have a long-running and slowly-progressing PR to refactor the test suite and introduce a class-based testing architecture that I think will enhance the comprehensibility and extensibility of the tests, and be useful for 3rd party packages after I have Cosmology entrance points implemented (another slowly-progressing PR). Having the ``flrw.X`` is convenient for find-and-replace!

Reply:
OK, makes sense!